clox/functions.py

```diff
@@ -85,6 +86,44 @@ def show_timezones_list():
         print("{0}. {1}".format(index, timezone))
 
 
+def print_calendar(mode="month", timezone=None, v_shift=0, h_shift=0):
+    """
+    Print calendar.
+
+    :param mode: calendar mode
+    :type mode: str
+    :param timezone: timezone
+    :type timezone: str
+    :param v_shift: vertical shift
+    :type v_shift: int
+    :param h_shift: horizontal shift
+    :type h_shift: int
+    :return: None
+    """
+    timezone_str = timezone
+    if timezone is None:
+        tz = None
+        timezone_str = "Local"
+    else:
+        tz = pytz.timezone(timezone)
+    v_shift = max(0, v_shift)
+    h_shift = max(0, h_shift)
+    datetime_now = datetime.datetime.now(tz=tz)
+    current_date = datetime_now.strftime(DATE_FORMAT)
+    print('\n' * v_shift, end='')
+    print(" " * h_shift, end='')
+    print("Today: {date}".format(date=current_date))
+    print(" " * h_shift, end='')
+    print("Timezone: {timezone}\n".format(timezone=timezone_str))
+    calendar_str = calendar.month(datetime_now.year, datetime_now.month)
+    if mode == "year":
+        calendar_str = calendar.calendar(datetime_now.year)
+    calendar_str_list = calendar_str.split("\n")
+    for index, line in enumerate(calendar_str_list):
+        calendar_str_list[index] = " " * h_shift + line
+    print("\n".join(calendar_str_list))
```

Comment:
That could be written more easily:
```pycon
print("\n".join([' ' * h_shift + x for x in calendar_str.split("\n")]))
```

Reply:
fixed in https://github.com/sepandhaghighi/clox/pull/34/commits/faa547d6633aeaf8af65ad02d88125c05bed6786